src/worker/account.ts

```diff
@@ -31,6 +31,12 @@ export async function updateAccount(
                 affectedAddresses.push(tx.pay!.receiver);
             }
         });
+        const unwrapCCCs = transactions.filter(tx => tx.type === "unwrapCCC");
+        unwrapCCCs.map(tx => {
```

Comment:
I think `forEach` is more appropriate than `map` here.

Reply:
I fixed.
And I made the filter filters only the successful transaction.